### PR TITLE
etherpad: add support for marathon provider

### PR DESCRIPTION
**Pull request by dustymabe**
and various other changes (each in its own commit)

### CODE REVIEW

File: etherpad-centos7-atomicapp/artifacts/kubernetes/etherpad-rc.yaml [hunk 1]
```diff
@@ -28,6 +28,6 @@ spec:
             - name: DB_PASS
               value: $db_pass
             - name: DB_PORT
-              value: '3306'
+              value: $db_port
```
Comment (kadel) [inline etherpad-centos7-atomicapp/artifacts/kubernetes/etherpad-rc.yaml:31]: Can you also add this to OpenShift artifacts?

Comment (dustymabe) [inline etherpad-centos7-atomicapp/artifacts/kubernetes/etherpad-rc.yaml:31]: done!

**Comment by kadel** [comment]
LGTM

File: etherpad-centos7-atomicapp/artifacts/kubernetes/etherpad-rc.yaml (hunk 1 above)
Comment (cdrage) [inline etherpad-centos7-atomicapp/artifacts/kubernetes/etherpad-rc.yaml:31]: this fix killed the kubernetes support for etherpad :( https://ci.centos.org/job/atomicapp-test-kubernetes-pr/141/console

**Pull request merged**